posts/templates/posts/post_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h3>Are you sure to delete?</h3>
```

Comment:
Just a sidenote, when asking for confirmation of anything, it's always good to include information about what the user will confirm. In this case you could write something like

    Are you sure to delete this post?
    <the post content>

so the user would know what they will delete.